pages/treeherder.py

```diff
@@ -51,6 +60,20 @@ def open_perfherder_page(self):
         from perfherder import PerfherderPage
         return PerfherderPage(self.base_url, self.selenium).wait_for_page_to_load()
 
+    def open_repos_menu(self):
+        self.selenium.find_element(*self._repos_menu_locator).click()
+
+    def select_random_repo(self):
+        self.selenium.open_repos_menu()
+        random.choice(unchecked_repos_links).click()
+        self.wait_for_page_to_load
+        return self.active_watched_repo
```

Comment:
We don't need to return the active watched repo, as the test can just call this directly.